app/Makefile

```
@@ -0,0 +1,150 @@
+# --- THIS IS PART OF GLOBAL MAKEFILE ---
+
+# Coverage reports for Python
+# Locally, use html report. In CI, xml:some/path
+PYTHON_COVERAGE_REPORT ?= html
+
+POETRY_WRAPPER_IMAGE_NAME ?= $(PROJECT_NAME)_poetry_wrapper
+
+
+.PHONY:
+poetry-lock:  ## Recreate Poetry lock file after any changes to pyproject.toml file
+	make _run-poetry-helper cmd="poetry lock $(args)"
+
+
+.PHONY:
+poetry-add: ## Add package to poetry: make poetry-add cmd=package
+	make _run-poetry-helper cmd="poetry add $(cmd)"
+
+
+.PHONY:
+fmt-py: autoflake black isort  ## Format all Python code
+
+
+.PHONY:
+lint-py: black-check isort-check flake8 pylint mypy  ## Lint all Python code
+
+
+.PHONY:
+quality-py: ## Run all Python code formatters and quality tools
+	make -k fmt-py flake8 pylint mypy
+
+
+.PHONY:
+test-py:  ## Run tests on Python code
+	docker-compose run python pytest $(args)
+
+.PHONY:
+coverage-py:  ## Runs python/django test coverage calculation
+	docker-compose run --rm -e COVERAGE_FILE=/home/user/.coverage python \
+		pytest --cov=. --cov-report $(PYTHON_COVERAGE_REPORT) --cov-report term-missing $(cmd)
+
+
+.PHONY:
+patch-coverage-py-paths:
+	sed -i -e "s|<source>.|<source>${CI_PROJECT_DIR}/${PROJECT_NAME}/|" $(PYTHON_COVERAGE_REPORT)
+
+
+.PHONY:
+autoflake:  args ?= .
+autoflake:  ## Run autoflake (fix unused imports and variables) on Python code
+	make -s run-python cmd="autoflake -r -i --remove-all-unused-imports --remove-unused-variables $(args)"
+
+
+.PHONY:
+black-check: args ?= .
+black-check:
+	make run-python cmd="black --check --diff $(args)"
+
+
+.PHONY:
+black: args ?= .
+black: ## Format Python code with Black
+	make -s run-python cmd="black $(args)"
+
+
+.PHONY:
+flake8: args ?= .
+flake8: ## Run Python code style checks
+	make -s run-python cmd='flake8 $(args)'
+
+
+.PHONY:
+pylint: ## Run Python code style checks
+ifdef args
+	make -s run-python cmd='pylint ${PROJECT_NAME} $(args)'
+else
+	make -s run-python cmd='pylint ${PROJECT_NAME}'
+endif
+
+.PHONY:
+mypy: args ?= .
+mypy:  ## Check Python types
+	echo -e "$(CYAN)Checking types$(COFF)"
+	make -s run-python cmd="mypy $(args)"
+
+
+.PHONY:
+isort-check:
+	echo -e "$(CYAN)Checking imports with isort$(COFF)"
+	make -s run-python cmd='isort tests $(PROJECT_NAME) --check --diff'
+
+
+.PHONY:
+isort: args ?= .
+isort: ## Fix imports automatically with isort
+	echo -e "$(CYAN)Fixing imports with isort$(COFF)"
+	make -s run-python cmd="isort $(args) --skip-glob='*/site-packages/*'"
+
+
+.PHONY:
+psql:
+	docker-compose exec postgres psql --user $(PROJECT_NAME) --dbname $(PROJECT_NAME)
```

```suggestion
	docker-compose exec postgres psql --user $(POSTGRES_USER) --dbname $(POSTGRES_DATABASE_NAME)
```
and need to include .env in Makefile.